pkg/oidcprovider/oidcprovider.go

```diff
@@ -350,6 +361,17 @@ func (p *provider) SetUserinfoFromScopes(ctx context.Context, userinfo *oidc.Use
 			userinfo.Email = user.Email
 			userinfo.EmailVerified = true
 		}
+
+		if strings.HasPrefix(scope, "pls_") {
```

Comment:
ditto, but maybe `GetUserPermissionsForGroup` handles the case where group is `""` fine? Haven't checked yet

Reply:
I added a check for it in `IsScopeAllowed`. If a client requests an invalid scope, it will implicitly be filtered out and not show up here.